lectures/getting_started.md

```diff
@@ -151,7 +155,7 @@ Either
 If you use the second option, you will see something like this
 
 ```{figure} /_static/lecture_specific/getting_started/starting_nb.png
-:scale: 60
+:figclass: terminal
 ```
```

Comment:
The only image I did not replace is the terminal view. It is because there are no significant changes in the terminal output.